TotalCrossVM/src/nm/lang/Runtime.c

```diff
@@ -6,6 +6,9 @@
 #include "Runtime.h"
 #if defined(HEADLESS)
 #include "cpproc.h"
+#include <sys/types.h>
```

Comment:
sys/types.h and errorno.h are inside and outside of the HEADLESS flag

Reply:
Should be only inside the HEADLESS flag, I'll fix it